lib/osf-components/addon/components/contributors/user-search/widget/component.ts

```diff
@@ -37,16 +38,22 @@ export default class UserSearchComponent extends Component<UserSearchComponentAr
         return this.currentUsersPage <= this.totalUsersPage;
     }
 
-    @task({ withTestWaiter: true, enqueue: true })
+    @task({ withTestWaiter: true, drop: true })
```

Comment:
For actual typing and then fetching `keepLatest ` is probably the better option.

Reply:
Actually, I just realized that we don't do search upon typing, only on clicking enter/search button.